documents/xaml-suggested-actions/xaml-designer-suggested-actions-extensibility.md

```diff
@@ -0,0 +1,110 @@
+## Suggested Actions Extensibility
+
+- [Tokens](./xaml-designer-suggested-actions-extensibility-tokens.md)
+- [Features](./xaml-designer-suggested-actions-extensibility-features.md)
+- [Behaviors](./xaml-designer-suggested-actions-extensibility-behaviors.md)
+- [Customization](./xaml-designer-suggested-actions-extensibility-customization.md)
+
+Starting from Visual Studio 2019 16.7 Preview 3 we added extensibility support for "XAML Suggested Actions"
+>*Note: this API will be available only in Preview Channel until feature is finalized and completed*
+
+To enable "Xaml Suggested Actions" for any control, `SuggestedActionProvider` feature provider should be created and registered in metadata.
+
+>*Control could have multiple Suggested Actions providers. Each separate provider will be shown as a separate tab in UI.*
```

Comment:
Control -> A control
Each separate -> Each
in UI -> "in the UI" or maybe something more specific like "in the Suggested Actions UI"?